Mathlib/FieldTheory/NormalClosure.lean

```diff
@@ -10,19 +10,35 @@ import Mathlib.Order.Closure
 #align_import field_theory.normal from "leanprover-community/mathlib"@"9fb8964792b4237dac6200193a0d533f1b3f7423"
 /-!
 # Normal closures
-The normal closure of a tower of fields `L/K/F` is the smallest intermediate field of `L/K` that
-contains the image of every `F`-algebra embedding `K →ₐ[F] L`.
-
-## Main Definitions
-- `normalClosure F K L` a tower of fields `L/K/F`.
+The normal closure `normalClosure F K L` of a field extension `K/F`
+in another field extension `L/F` is the smallest intermediate field of `L/K`
+that contains the image of every `F`-algebra embedding `K →ₐ[F] L`.
+
+If the minimal polynomial of every element of `K` over `F` splits in `L` (in particular if `L/K/F`
+form a tower and `L/F` is normal), then `normalClosure F K L` is a valid normal closure of `K/F`,
+characterized by the predicate `IsNormalClosure F K L`, which says that `L` is generated
+by the roots of the minimal polynomials in addition to the splitting condition.
 -/
 
-open BigOperators IntermediateField IsScalarTower Polynomial
+universe u v w
 
-variable (F K L : Type*) [Field F] [Field K] [Field L] [Algebra F K] [Algebra F L] [Algebra K L]
-  [IsScalarTower F K L]
+open BigOperators IntermediateField IsScalarTower Polynomial
 
-/-- The normal closure of `K` in `L`. -/
+variable (F : Type*) (K : Type w) (L : Type u)
```

Comment:
Is there a reason for preferring explicit universes here?

Reply:
The universes are for the statement of `cardinal_mk_algHom_le_of_splits` but I guess I'll replace that with an Embedding.